packages/@storefront/structure/src/filtered-list/index.ts

```diff
@@ -47,6 +49,12 @@ class FilteredList {
     }
   }
 
+  onSelectAll = (event: Tag.Event) => {
```

Comment:
Is there a reason to use a function expression rather than a method definition here? This approach makes it slightly more difficult for a derived component class to shadow this function.

Reply:
This will be addressed in ticket SF-1108.

Reply:
Sounds good; resolving.